### PR TITLE
Option to allow to control the AgentForwarding setting for ssh_config

### PR DESCRIPTION
**Pull request by rkno82**
Hi,

to ensure our private keys by not spreading that keys to multiple jumphosts, we need that option.
I've adjusted what needs to be done.

fingers crossed for the tests :)

Thanks,
Robert

### CODE REVIEW

**Comment by rkno82** [comment]
@schurzi @rndmh3ro @neubi4 can you check please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 ## [UNRELEASED]
 
 ## [2.6.0] - 2018-07-19
-* initial public release
\ No newline at end of file
+* initial public release
+
+## [2.7.0] - 2018-10-16
```
Comment (schurzi) [inline CHANGELOG.md]: please reverse order, newest comes first

File: manifests/init.pp [hunk 2]
```diff
@@ -15,6 +15,7 @@
   $ext_sshd_AuthorizedKeysCommand           = undef,
   $ext_sshd_AuthorizedKeysCommandUser       = 'nobody',
   $ext_sshd_ChallengeResponseAuthentication = 'no',
+  $ext_ssh_ForwardAgent                     = 'no',
```
Comment (schurzi) [inline manifests/init.pp]: please add one extra test, where you modify the default value for $ext_ssh_ForwardAgent

**Comment by rndmh3ro** [comment]
LGTM

**Comment by schurzi** [comment]
please add this snipet to .travis.yml or else deployment will fail:
```
before_deploy:
- rvm $(travis_internal_ruby) --fuzzy do ruby -S gem install puppet -v 5.5.6
```

(compare: https://github.com/T-Systems-MMS/puppet-secc_snmpd/blob/master/.travis.yml)

**Comment by rkno82** [comment]
> 
> 
> please add this snipet to .travis.yml or else deployment will fail:
> 
> ```
> before_deploy:
> - rvm $(travis_internal_ruby) --fuzzy do ruby -S gem install puppet -v 5.5.6
> ```
> 
> (compare: https://github.com/T-Systems-MMS/puppet-secc_snmpd/blob/master/.travis.yml)

Worked with and without adjustment: 

https://travis-ci.org/T-Systems-MMS/puppet-secc_sshd/builds/442644718?utm_source=github_status&utm_medium=notification

**Pull request merged**